app/tests/unit/lib/theme/CustomDiffFields.test.tsx

```diff
@@ -77,7 +77,8 @@ const latestCommittedData = {
 };
 
 describe("The Object Field Template", () => {
```

Comment:
Test name could use an update

I think we're missing a few tests still:
- if there have been no changes to anything, we should see no diffs
- if we have old and new but not latest committed
- arrows show up correctly